sdk/identity/identity/src/credentials/interactiveBrowserCredential.ts

```diff
@@ -137,14 +137,18 @@ export class InteractiveBrowserCredential implements TokenCredential {
         try {
           const authResponse = await this.msalClient.acquireTokenByCode(tokenRequest);
           const successMessage = `Authentication Complete. You can close the browser and return to the application.`;
-          const expiresOnTimestamp = authResponse?.expiresOn.valueOf();
-          res.status(200).send(successMessage);
-          logger.getToken.info(formatSuccess(scopeArray));
-
-          resolve({
-            expiresOnTimestamp,
-            token: authResponse.accessToken
-          });
+          if (authResponse && authResponse.expiresOn) {
+            const expiresOnTimestamp = authResponse?.expiresOn.valueOf();
+            res.status(200).send(successMessage);
+            logger.getToken.info(formatSuccess(scopeArray));
+
+            resolve({
+              expiresOnTimestamp,
+              token: authResponse.accessToken
+            });
+          } else {
+            reject(new Error("Did not receive a token with a valid expiration"));
```

Comment:
In DeviceCodeCredential, we use "Did not receive token with timestamp" where as here we use "Did not receive a token with a valid expiration". Both for the case when we either get a null response or a response with no expiry date. 

Another difference is that in DeviceCodeCredential, we prefix the error with `Device Authentication Error`. Any reason for no prefixes in InteractiveBrowserCredential?